rust_snuba/src/strategies/clickhouse/batch.rs

```diff
@@ -48,7 +50,14 @@ impl BatchFactory {
             HeaderValue::from_str(database).unwrap(),
         );
 
-        let query_params = "load_balancing=in_order&insert_distributed_sync=1".to_string();
+        let mut query_params = String::new();
+        query_params.push_str("load_balancing=in_order&insert_distributed_sync=1");
+
+        let async_inserts_allowed = get_str_config("async_inserts_allowed").ok().flatten();
+        if async_inserts == true && async_inserts_allowed == Some(1.to_string()) {
```

Comment:
I don't know if this is the right way to do this, but my goal is to have a runtime config that we can turn on and off for our testing purposes as well as a cli arg (`async_inserts`) for the specific cluster we want

This generic runtime config wouldn't really scale if we are trying to turn async inserts on and off for different clusters, but it's mainly for running the first experiments in a safe/easy way

Reply:
this looks fine, however i would suggest to only query the get_str_config when `async_inserts` is true